docs/templates/plugin.rst.j2

```diff
@@ -148,52 +178,65 @@ Common return values are documented here :doc:`common_return_values`, the follow
 .. raw:: html
 
     <table border=1 cellpadding=4>
+
     <tr>
     <th class="head">name</th>
     <th class="head">description</th>
     <th class="head">returned</th>
     <th class="head">type</th>
     <th class="head">sample</th>
     </tr>
+{% for entry in returndocs %}
 
-    {% for entry in returndocs %}
     <tr>
-        <td> @{ entry }@ </td>
-        <td> @{ returndocs[entry].description }@ </td>
-        <td align=center> @{ returndocs[entry].returned }@ </td>
-        <td align=center> @{ returndocs[entry].type }@ </td>
-        <td align=center> @{ returndocs[entry].sample}@ </td>
+    <td>@{ entry }@</td>
+    <td>
+{% for desc in returndocs[entry].description %}
+        <div>@{ desc | replace('\n', '\n    ') | html_ify }@</div>
+{% endfor %}
```

Comment:
The schema defines this field as either a string or a list of strings.  Wouldn't this break the instance where it is just a string?  We would be iterating each individual character of that string.